api-client/src/endpoints/__tests__/createDisclaimerEndpoint.spec.ts

```diff
@@ -9,7 +10,7 @@ describe('disclaimer', () => {
     title: 'Feedback, Kontakt und mögliches Engagement',
     type: 'disclaimer',
     modified_gmt: '2017-06-12 12:27:57',
-    content: '<a href="javascript:IWantToBeRemoved();">Ich bleib aber da.</a>',
+    content: '<div>Some disclaimer test content :)</div>',
```

Comment:
I think removing the javascript makes the test invalid. I think this test is now only testing that the content is not changing.